charts/rancher-turtles/templates/post-delete-job.yaml

```diff
@@ -122,7 +126,7 @@ spec:
         - delete
         - deployments.apps/capi-controller-manager
         - -n
-        - capi-system
+        - {{ index .Values "cluster-api-operator" "cluster-api" "core" "namespace" }}
```

Comment:
can we do the same for kubeadm-cpl/bootstrap namespaces (lines 138,147)?

Reply:
I don't know hot to handle it, we removed values for kubeadm namespace

Reply:
checked and they fine as is and can't be fetched